### ISSUE TITLE
Format documents while Dendron doctor command is performed

### ISSUE THREAD

**Issue by grizz-pl** [dendronhq/dendron#503]
## Please Select if your Request is Either something new or an Enhancement ##

- [x] Enhancement of an existing Feature.
- [ ] Request of a new feature.

## Please select the area your request applys to. (Multiple selections are Possible) ##

- [ ] UI
- [x] Workbench
- [ ] Publishing
- [ ] Pods
- [ ] Markdown
- [ ] Hierarchy
- [ ] Graph
- [ ] Misc (Choose this if your not sure)

It would be nice to have an option to format all documents (VSCODE command Format document) while Dendron doctor command is performed.

**Comment by kevins8** [comment]
Thanks for the ticket. So it might not look like it but Dendron Doctor will
format documents that need fixing. Have a task to break out Doctor actions
so you can choose to just apply formatting.

**Comment by grizz-pl** [comment]
What does mean 'Dendron Doctor will
format documents that need fixing'? I would like to have a possibility to use VSCODE command - so I can appply things like align table collumns.

**Comment by kevinslin** [comment]
Doctor works on notes that have issues (eg. missing frontmatter, lacks title, etc). Currently, formatting occurs as a side effect. We don't currently have a dedicated format operation. Will add this to our backlog. Also related to #278

**Comment by kpathakota** [comment]
Yeah, this is a good suggestion though couldn't this be done already using an extension like [prettier](https://stackoverflow.com/questions/43666270/how-do-i-format-all-files-in-a-visual-studio-code-project)
